webapp/src/js/services/live-list.js

```diff
@@ -521,6 +536,7 @@ angular.module('inboxServices').factory('LiveList',
         initialised: _.partial(_initialised, name),
         setSelected: _.partial(_setSelected, name),
         clearSelected: _.partial(_clearSelected, name),
+        isContainedDeleteStub: _.partial(_isContainedDeleteStub, name)
```

Comment:
I'm not sure about the name here. Maybe it should be `containsDeleteStub`?

Reply:
Done.